lib/js/review/SubmitController.js

```diff
@@ -138,7 +138,7 @@ var SubmitController = function (_React$Component) {
         ),
         _react2.default.createElement(_PrivacyAgreement2.default, {
           required: true,
-          onChange: this.props.setPrivacyAgreement,
+          onChange: _actions.setPrivacyAgreement,
           checked: privacyAgreementAccepted,
```

Comment:
This looks like a result of me not doing a `npm run build` on PR #89 right? The CI will be doing that automatically but we may also need a commit hook so that the built files don't get out of sync with the original source.

Reply:
Yes, I think so. A commit hook is a good idea. I'll create a ticket for that.